src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -2738,6 +2738,53 @@ public void TestFTSQueryWithJoin()
             }
         }
 
+        [Fact]
+        public void TestSelectAllResultKey()
+        {
+            // Section 8.11.10 / 8.12.4 - CBL Helium - Scope and Collection API
+            using var c = Db.CreateCollection("flowers", "test");
+            using var d = Db.GetDefaultCollection();
+
+            var iterations = new List<(string queryID, string resultName)>

```

Comment:
This really should be called "expectedResultNames" or something halfway intelligible.
 Same with qbIterations

Reply:
Updated!